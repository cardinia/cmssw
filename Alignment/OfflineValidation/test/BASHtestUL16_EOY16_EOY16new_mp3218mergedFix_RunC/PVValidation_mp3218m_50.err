27-Sep-2019 21:42:51 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016C/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/0AD9CD57-CD07-E911-968B-AC1F6B0DE362.root
27-Sep-2019 21:42:55 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016C/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/0AD9CD57-CD07-E911-968B-AC1F6B0DE362.root
Begin processing the 1st record. Run 276095, Event 21322, LumiSection 1 on stream 0 at 27-Sep-2019 21:43:01.079 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 21:43:56 CEST  Run: 276095 Event: 1810316
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 126
%MSG
27-Sep-2019 21:43:57 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016C/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/0AD9CD57-CD07-E911-968B-AC1F6B0DE362.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       1        1
    2 fileAction           -s file_close                             1        1
    3 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          276095/1810316                    
    2 fileAction           PostGlobalEndRun                  
    3 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 1                   1
System                  3                   3

dropped waiting message count 0
[7.008MB/7.008MB][100%][==================================================][7.008MB/s]  [7.008MB/7.008MB][100%][==================================================][7.008MB/s]  
