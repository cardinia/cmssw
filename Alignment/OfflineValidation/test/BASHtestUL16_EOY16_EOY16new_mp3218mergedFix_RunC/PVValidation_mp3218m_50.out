Job started at  Fri Sep 27 21:42:34 CEST 2019
batch dir: /pool/condor/dir_2663 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_276095_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 7.1M -rw-r--r--. 1 tkello zh 7.1M Sep 27 21:44 PVValidation_mp3218m_276095.root -rw-r--r--. 1 tkello zh 20K Sep 27 21:42 PVValidation_mp3218m_276095_cfg.py -rw-r--r--. 1 tkello zh 1.9K Sep 27 21:44 _condor_stderr -rw-r--r--. 1 tkello zh 521 Sep 27 21:42 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 27 21:42 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 21:42 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:42 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:42 var
Job ended at  Fri Sep 27 21:44:23 CEST 2019
